tests/testExecutables.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+import lsst.utils.tests
+
+
+class UtilsBinaryTester(lsst.utils.tests.ExecutablesTestCase):
+    pass
+
+EXECUTABLES = ['testPsfAttributes', 'testSizeMagnitudeStarSelector', 'testImagePsf', 'testPsfCaching', 'testWarpedPsf']
```

Comment:
I didn't know it did that! It is great if it does that. I can check and revert to the default if that's true...